tagged-localbuild.xml

```diff
@@ -60,7 +60,7 @@
   <project name="kernel/configs" revision="e4102999005f8f3c8db1fdacfba80a2c581a72f1" upstream="refs/tags/android-11.0.0_r43" dest-branch="refs/tags/android-11.0.0_r43" groups="vts,pdk"/>
   <project name="macaddrsetup" path="vendor/oss/macaddrsetup" remote="sony" revision="d2871b2041790f6d291e207bec8b0ef34029ef59" upstream="master" dest-branch="master" groups="device"/>
   <project name="nfc-NXPNFCC_FW" path="vendor/nxp/NXPNFCC_FW" remote="NXP" revision="08f462368aa9cfd721172cc1facc00a6d325a386" upstream="master" dest-branch="master" groups="device"/>
-  <project name="packages_apps_ExtendedSettings" path="packages/apps/ExtendedSettings" remote="sony" revision="0973b752827008710b5b4ff30128affca34d22e3" upstream="master" dest-branch="master" groups="device"/>
+  <project name="packages_apps_ExtendedSettings" path="packages/apps/ExtendedSettings" remote="sony" revision="0973b752827008710b5b4ff30128affca34d22e3" upstream="master" dest-branch="r-mr1" groups="device"/>
```

Comment:
this shouldn't belong to this commit (and is later reverted and re-added in the subsequent commits)